src/aa_api_wrapper/aleph_alpha.py

```diff
@@ -0,0 +1,70 @@
+import os
+from aleph_alpha_client import Client, CompletionRequest, Prompt
+from aleph_alpha_client.embedding import (
+    SemanticEmbeddingRequest,
+    SemanticRepresentation,
+)
+from fastapi import Request
+from openai.types.completion_create_params import CompletionCreateParams
+from typing import Literal
+
+from openai.types.embedding_create_params import EmbeddingCreateParams
+
+from aa_api_wrapper.http import unpack_bearer_token
+
+
+AaFinishReason = Literal["end_of_text", "maximum_tokens"] | None
+
+
+def init_client(token: str) -> Client:
+    aa_client = Client(
+        host=os.environ.get("ALEPH_ALPHA_API_BASE", "https://api.aleph-alpha.com"),
+        nice=True,
+        token=token,
+    )
+    return aa_client
+
+
+def init_from_request(request: Request):
+    token = unpack_bearer_token(request)
+    return init_client(token)
+
+
+def create_embedding_request(
+    embedding_params: EmbeddingCreateParams,
+) -> SemanticEmbeddingRequest:
+    embedding_input = embedding_params["input"]
+    assert isinstance(
+        embedding_input, str
+    ), "Lists of inputs are currently not supported"
+
+    prompt = Prompt.from_text(embedding_input)
+    return SemanticEmbeddingRequest(
+        prompt=prompt, representation=SemanticRepresentation.Document
+    )
+
+
+def create_completion_request(
+    completion_params: CompletionCreateParams,
+) -> CompletionRequest:
+    text = completion_params["prompt"]
+    assert isinstance(text, str), "Lists of prompts are currently not supported"
+    prompt = Prompt.from_text(text)
+
+    stop = completion_params.get("stop")
+    stop_sequences = stop if isinstance(stop, list) else [stop] if stop else None
+
+    return CompletionRequest(
+        prompt=prompt,
+        maximum_tokens=completion_params.get("max_tokens"),
+        temperature=completion_params.get("temperature") or 0.0,
```

Comment:
Why don't we use the `default` field of `get`? Sth like `get("temperature", 0.0)`?